src/components/Request/index.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react'
+import {startsWith} from 'lodash'
+import {logout} from 'utils/authentication'
+import {default as RequestBase} from './components/Request'
```

Comment:
maybe rename the actual subdirectory to be `RequestBase` since that's what you're importing as? I admit to being kind of confused at first when reading two different `Request/index.js` files back to back, even though I know from the README that there are going to be subfolders for subcomponents.